clients/src/main/java/org/apache/kafka/common/security/JaasUtils.java

```diff
@@ -50,6 +50,9 @@ public static String zkSecuritySysConfigString() {
     }
 
     public static boolean isZkSecurityEnabled() {
```

Comment:
Should we rename this method to `isZkSaslEnabled()`or  something along those lines?